pytorch_lightning/accelerators/__init__.py

```diff
@@ -14,4 +14,9 @@
 from pytorch_lightning.accelerators.cpu import CPUAccelerator  # noqa: F401
 from pytorch_lightning.accelerators.gpu import GPUAccelerator  # noqa: F401
 from pytorch_lightning.accelerators.ipu import IPUAccelerator  # noqa: F401
+from pytorch_lightning.accelerators.registry import AcceleratorRegistry, call_register_accelerators  # noqa: F401
 from pytorch_lightning.accelerators.tpu import TPUAccelerator  # noqa: F401
+
+ACCELERATORS_BASE_MODULE = "pytorch_lightning.accelerators"
```

Comment:
Why expose this string?

If it was to avoid registering these by default, it would not work because to change the path you'd need to import the variable

And at import time, they would get registered anyways.